libs/sketch/src/lib/components/dialog/dialog.component.stories.ts

```diff
@@ -99,7 +97,7 @@ Default.argTypes = {
     description: 'the position of the close button',
     defaultValue: { summary: CloseButtonPosition.Right },
   },
-};
+} satisfies ArgTypes;
```

Comment:
is this to get rid of the as .. as ..? if yes this is great to know! I really didnt liked it. 
If not, what is it for?